crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstner.cs

```diff
@@ -323,6 +330,12 @@ public void CrestUpdate(CommandBuffer buf)
             // Seems like shader errors cause this to unbind if I don't set it every frame. Could be an editor only issue.
             _matGenerateWaves.SetTexture(sp_WaveBuffer, _waveBuffers);
 
+            if (_meshForDrawingWaves != null && _matGenerateWavesGeometry != null)
```

Comment:
Does this need to be done every frame? If so its not the end of the world, just checking. (Would wrapping in `#if UNITY_EDITOR` potentially work?)

Reply:
Only on setup I think. Done.